rotary_bed_MO.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+import pandas as pd
+import pyomo.environ as pyo
+from measure_optimize import (
+    MeasurementOptimizer,
+    SensitivityData,
+    MeasurementData,
+    CovarianceStructure,
+    ObjectiveLib,
+)
+import pickle
+import time
+
+
+# set up problem formulation
+
+# number of time points for DCM
+Nt = 110
+# maximum manual measurement number for each measurement
+max_manual_num = 5
+# minimal measurement interval
+min_interval_num = 10.0
+# maximum manual measurement number for all measurements
+total_max_manual_num = 20
+# index of columns of SCM and DCM in Q
+static_ind = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
First, declare the names, then declare the indices. This will be easier to understand.

Reply:
added